### ISSUE TITLE
incorporate recordGroupSample into GA4GH schema

### ISSUE THREAD

**Issue by akmorrow13** [bigdatagenomics/convert#53]
Currently, we use recordGroupName as readGroupId, but recordGroupSample is not put in the schema.

**Comment by heuermh** [comment]
I don't see anywhere to put it, have any ideas?

**Comment by akmorrow13** [comment]
I am not sure. @jpdna  suggested combining recordGroupSample and recordGroupName into readGroupId, but we still have the same issue of not being able to identify samples based on readGroupId.  It would, however, better identify the sample in readGroupId.

**Comment by heuermh** [comment]
It's hard to follow ga4gh, since the schemas have all been deprecated out of existence.  That said, there are separate `ReadGroup` and `ReadGroupSet` message schema, but they are not directly referenced from `ReadAlignment`.  We'd have to create our own tuples or another model that would do the join across readGroupId.
